chain-configs/chains.json

```diff
@@ -0,0 +1,595 @@
+{
+  "akash": {
```

Comment:
Can we maintain separate file for each network? Maintaining separate files for each network makes it easier to add support for new network without constantly updating a single file.